yarn.lock

```diff
@@ -2,152 +2,147 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.5.5":
-  version "7.5.5"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.5.5.tgz#bc0782f6d69f7b7d49531219699b988f669a8f9d"
-  integrity sha512-27d4lZoomVyo51VegxI20xZPuSHusqbQag/ztrBC7wegWoQ1nLREPVSKSW8byhTlzTKyNE4ifaTA6lCp7JjpFw==
-  dependencies:
-    "@babel/highlight" "^7.0.0"
-
-"@babel/code-frame@^7.8.3":
+"@ampproject/remapping@0.2.0":
+  version "0.2.0"
+  resolved "https://registry.yarnpkg.com/@ampproject/remapping/-/remapping-0.2.0.tgz#07290a5c0f5eac8a4c33d38aa0d15a3416db432e"
+  integrity sha512-a4EztS9/GOVQjX5Ol+Iz33TFhaXvYBF7aB6D8+Qz0/SCIxOm3UNRhGZiwcCuJ8/Ifc6NCogp3S48kc5hFxRpUw==
+  dependencies:
+    "@jridgewell/resolve-uri" "1.0.0"
+    sourcemap-codec "1.4.8"
+
+"@ampproject/rollup-plugin-closure-compiler@^0.22.2":
+  version "0.22.2"
+  resolved "https://registry.yarnpkg.com/@ampproject/rollup-plugin-closure-compiler/-/rollup-plugin-closure-compiler-0.22.2.tgz#7b26fab6db7a08d01e9922209ab622f82e0f991c"
+  integrity sha512-loT6IJyJ8wBLAomlTW279OZgxDB/aR7pOTmtPBm4x4GZVjN+E15TEbgCohgYp2bWfNDcwVzRp30qGQ/WiJelEA==
+  dependencies:
+    "@ampproject/remapping" "0.2.0"
+    "@kristoferbaxter/estree-walker" "2.0.0"
+    acorn "7.1.0"
+    acorn-walk "7.1.1"
+    google-closure-compiler "20200204.0.0"
+    magic-string "0.25.6"
+    uuid "3.4.0"
+
+"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.5.5", "@babel/code-frame@^7.8.3":
   version "7.8.3"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.8.3.tgz#33e25903d7481181534e12ec0a25f16b6fcf419e"
   integrity sha512-a9gxpmdXtZEInkCSHUJDLHZVBgb1QS0jhss4cPP93EW7s+uC5bikET2twEF3KV+7rDblJcmNvTR7VJejqd2C2g==
   dependencies:
     "@babel/highlight" "^7.8.3"
 
+"@babel/compat-data@^7.8.6", "@babel/compat-data@^7.9.0":
```

Comment:
Something definitely went wrong with the merge conflict resolution here; [your initial commit](https://github.com/jaredpalmer/tsdx/pull/525/commits/a17e146f9f3ac637cb285bb029942d740153a75b) has a **10x** smaller `yarn.lock` diff.

When resolving conflicts with lockfiles like this, I'd recommend throwing out the changes you have in the lockfile and just re-generating the lockfile changes with `yarn`. So take what was added in the conflict and then run `yarn` on top of it to re-add your changes.